containers-create-client/story/en-US.json

```diff
@@ -190,13 +237,13 @@
         "INCLUDES": "INCLUDES",
         "DELETE": "Delete",
         "DELETE_FROM_LIST": "Are you sure you want to delete this template?",
-        "CREATING_IN_PLAYBUZZ": "Creating in Playbuzz just became easier!",
+        "CREATING_IN_PLAYBUZZ": "You have no templates",
```

Comment:
Can you explain this?

Reply:
@iddanh text is according to design

Reply:
It's just that the key doesn't match the value